lms/lib/courseware_search/lms_result_processor.py

```diff
@@ -77,7 +81,8 @@ def get_position_name(section, position):
             section_item = self.get_item(course_key.make_usage_key("sequential", section))
             if section_item.has_children and len(section_item.children) >= pos:
                 item = self.get_item(section_item.children[pos - 1])
-                return getattr(item, "display_name", None)
+                display_name = getattr(item, "display_name", None)
```

Comment:
Rather than referencing a shared param, what about moving this duplicate block into a helper method and call it here and on line 75